projects/plugins/jetpack/modules/subscriptions/subscribe-modal/subscribe-modal.css

```diff
@@ -0,0 +1,45 @@
+#wpcom-subscribe-modal {
+	visibility: hidden;
+	position: fixed;
+	z-index: 1;
+	left: 0;
+	top: 0;
+	width: 100%;
+	height: 100%;
+	overflow: auto;
+	background-color: transparent;
+	transition: all 0.4s;
+}
+
+#wpcom-subscribe-modal.open {
+	background-color: rgba(0,0,0,0.2);
+	visibility: visible;
+}
+
+#wpcom-subscribe-modal .modal-content {
+	position: relative;
+	visibility: hidden;
+    top: 100%;
+	background-color: #fefefe;
```

Comment:
Some mixed tabs/spaces white spacing here and elsewhere in the file.

Reply:
Fixed!